models/prices/prices_tokens.sql

```diff
@@ -509,6 +509,7 @@ VALUES
     ("ohmv2-olympus-v2", "ethereum", "OHM", "0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5", 9),
     ("okb-okb", "ethereum", "OKB", "0x75231f58b43240c9718dd58b4967c5114342a86c", 18),
     ("ole-openleverage", "ethereum", "OLE", "0x92cfbec26c206c90aee3b7c66a9ae673754fab7e", 18),
+    ("ole-openleverage", "ethereum", "OLE", "0xa865197a84e780957422237b5d152772654341f3", 18),
```

Comment:
this address appears to tie to bnb chain -- does the chain need edited here to reflect that rather than ethereum?
https://bscscan.com/token/0xa865197a84e780957422237b5d152772654341f3